src/core/Footer/Footer.test.tsx

```diff
@@ -45,6 +45,10 @@ const customUrls: FooterElement[] = [
   },
 ];
 
+const customUrlsFunction = (defaultElements: FooterElement[]) => {
```

Comment:
How did you manage to push with linter warnings? I thought we have a hook for that :/

Reply:
Fixed